src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
+import { User } from '../../Types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <a
+      className="UserInfo"
+      href={`mailto:${email}`}
+    >
+      {name}
+    </a>
+  );
```

Comment:
You could write it like this, but I like the way you done it also
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => (
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
);
```